### ISSUE TITLE
Cant get the reported accuracy by running the training scripts.

### ISSUE THREAD

**Issue by Amose-Yao** [aamir-mustafa/pcl-adversarial-defense#10]
Hi,
I am impressed by your work. But when I run the scripts which are softmax_training.py and pcl_training.py, I only got the 91 and 78 respectively. I wonder there could be some differences between your configuration and mine. Could you please share the hardware configuration and the hyperparameters with us?
Best wishes and Merry Christmas,
Amose.

**Comment by aamir-mustafa** [comment]
Hi,

Yes I am planning to share a separate file containing the hyper-parameters chosen while training for all 5 datasets used in the paper.
Meanwhile, I would suggest reducing weight_prox in pcl_training.py. This should help in improving the clean accuracy of the model.

Hope it helps
